R/function-body.R

```diff
@@ -0,0 +1,109 @@
+setOldClass("rxUi")
+
+bodySetRxUi <- function(fun, envir = environment(fun), value) {
+  if (is.function(value)) {
+    value <- body(value)
+  }
+  dropEnv <-
+    c(
```

Comment:
These are hard coded.... I don't think they should be, it should be a white list not a black list

Reply:
Or should be allowed to be removed by some heuristic.

Reply:
@mattfidler, I don't know what the correct list should be.  I copied this from nlmixr2/nlmixr2lib#33.  If you have a better way to do it, can you please let me know what that is specifically (or make a commit to this PR to do it)?